airbyte-db/lib/src/main/resources/configs_database/schema_dump.txt

```diff
@@ -1,3 +1,42 @@
+create table "public"."actor"(
+  "id" uuid not null,
+  "workspace_id" uuid not null,
+  "actor_definition_id" uuid not null,
+  "name" varchar(256) not null,
+  "configuration" jsonb not null,
+  "actor_type" actor_type not null,
+  "tombstone" bool not null,
+  "created_at" timestamptz(35) not null,
```

Comment:
for all of the created_at and updated_at can we have it default to the current timestamp, please? setting that default helps avoid a bunch of manual input that would otherwise just be the current timestamp anyway.